### PR TITLE
Add blackbird media player component

### PR DESCRIPTION
**Pull request by koolsb**
**Description:**

Documentation for new Monoprice Blackbird component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#13549

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/media_player.blackbird.markdown [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+The `blackbird` platform allows you to control [Monoprice Blackbird Matrix Switch](https://www.monoprice.com/product?p_id=21819) using a serial connection.
+
+To add a Blackbird device to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: blackbird
+    type: serial
```
Comment (frenck) [inline source/_components/media_player.blackbird.markdown:24]: Please keep the initial examples as minimal as possible.
- Remove optional parameters
- Keep (required) lists a minimal as possible.

If you'd like, you could provide an additional and more extensive example (e.g., at the end of the page).

**Review by frenck — CHANGES_REQUESTED** [review]
Looks good in general 👍 

Left you a comment about the example yaml. Could you please take a look @koolsb?

File: source/_components/media_player.blackbird.markdown (hunk 1 above)
Comment (koolsb) [reply source/_components/media_player.blackbird.markdown:24]: Thanks @frenck! I updated the example yaml as requested.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks for updating @koolsb! ✅ 

Can be merged as soon as the parent PR gets merged.

**Review by frenck — APPROVED** [review]

**Pull request merged**